scheduler/src/cook/mesos/api.clj

```diff
@@ -1555,6 +1557,10 @@
                                                   (:uuid %)
                                                   []))
                           groups)]
+
+      (let [user (get-in ctx [:request :authorization/user])]
+        (rt/spend-tokens! job-submission-rate-limiter user (count jobs)))
```

Comment:
If token bucket filters are just one possible implementation of rate limiting, doesn't having the word `tokens` in the abstraction introduce a leak down to the implementation? Should this just be `spend!`? Or `spend-resources!`? Or `use-resources!`?